extensions/adobe/experience/decisioning/calendar-constraint-details.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/decisioning/calendar-constraint-details",
+  "meta:extensible": true,
+  "type": "object",
+  "title": "Calendar Constraint Details",
+  "description": "A calendar constraint is a component of a decision option that defines the date range when option is valid. Outside that date range the option cannot be proposed.",
+  "definitions": {
+    "calendar-constraint-details": {
+      "type": "object",
+      "properties": {
+        "xdm:startDate": {
```

Comment:
Given the namespace of this new component, let's name this field https://ns.adobe.com/experience/decisioning/startDate

Reply:
This is a datatype schema (ie. a separate schema file) and using those schemas for properties in mixins or other datatype schemas requires to use a property prefix that gets erased. Otherwise I see doubly nested paths like "_experience.decisioning.myProp._experience.decisioning.datatypeProp1"

Reply:
In AEP it will be _experience.decisioning.myProp.datatypeProp1. I understand the XED validator in the PR check is not doing this, but it will in AEP. 

The goal here is to not have XDM namespaced components sitting out in the extension folder. If you need it to be XDM that is fine, but let's put those under the standard components/datatypes location then.